app/src/main/res/layout/activity_top_navigation.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/drawer_layout"
+    android:id="@+id/top_navigation_drawer_layout"
```

Comment:
Not a fan of this id naming as we'll be using DataBinding to reference views by their Ids. This type of naming will make it harder so best to go without the `activity_name` prefix